electrum/lnpeer.py

```diff
@@ -1769,29 +1769,69 @@ async def _shutdown(self, chan: Channel, payload, *, is_local: bool):
         # estimate fee of closing tx
         our_sig, closing_tx = chan.make_closing_tx(our_scriptpubkey, their_scriptpubkey, fee_sat=0)
         fee_rate = self.network.config.fee_per_kb()
-        our_fee = fee_rate * closing_tx.estimated_size() // 1000
+        our_fee = fee_rate * closing_tx.estimated_size() // 1000  # inclusion in one block
```

Comment:
Actually, no, `fee_rate` is not defined as inclusion in one block. It is the fee rate resulting from user preferences.
Note that it differs from the fee rate used in commitment transactions. which uses a dynamic fee with ETA of 2 blocks.